frontend/src/AlertConfiguration/AlertConfigurationComponents/AlertConfigurationApp.js

```diff
@@ -0,0 +1,36 @@
+import React, { Fragment }from 'react';
```

Comment:
nit: space between the } and from
also rename this file to AlertConfiguration.js please!